app/routes/member.js

```diff
@@ -1,7 +1,26 @@
 import AuthenticatedRoute from 'ghost-admin/routes/authenticated';
 
 export default AuthenticatedRoute.extend({
-    // TODO: add model method to load member if not passed in
+    model(params) {
+        this.set('isMemberUpdated', true);
```

Comment:
This is more of an FYI. Generally we try to limit use of `.set()` to when a property is going to be used in a template or another computed property where tracking is needed. In this case it's completely internal to the route instance so `this.isMemberUpdated = true` would be preferable I think, or even `this._isMemberUpdated = true` to indicate that this is a "private" property.

The more we can standardise things now the easier it will be to switch to newer Ember/esnext idioms such as [tracked properties](https://blog.emberjs.com/2019/02/26/coming-soon-in-ember-octane-part-3.html) and real [private properties on classes](https://github.com/tc39/proposal-class-fields#private-fields).

Reply:
💯 agree, that sounds pretty sane to me. Its also good to know the right usage or best practices wherever possible in PRs, so please keep them coming 😄 I'll update here as well asap